src/simularium/VisDataCache.ts

```diff
@@ -111,7 +138,10 @@ class VisDataCache {
     }
 
     public getFirstFrameNumber(): number {
-        return this.head?.data.frameNumber || -1;
+        if (this.head !== null) {
+            return this.head.data.frameNumber;
+        }
+        return -1;
```

Comment:
If the issue here was that `0` was getting ignored, the solution might be the `??` operator?